src/main/java/com/google/devtools/build/lib/analysis/test/TestConfiguration.java

```diff
@@ -84,6 +86,24 @@ public static class TestOptions extends FragmentOptions {
                 + "-1 tells blaze to use its default timeouts for that category.")
     public Map<TestTimeout, Duration> testTimeout;
 
+    @Option(
+        name = "test_resources",
+        defaultValue = "null",
+        converter = TestResourcesConverter.class,
+        allowMultiple = true,
+        documentationCategory = OptionDocumentationCategory.TESTING,
+        effectTags = {OptionEffectTag.UNKNOWN},
+        help =
+            "Override the default resources amount for tests. The expected format is"
+                + " <resource>=<value>. If a single positive number is specified as <value>"
+                + " it will override the default resources for all test sizes. If 4"
+                + " comma-separated numbers are specified, they will override the resource"
+                + " amount for respectively the small, medium, large, enormous test sizes."
+                + " Values can also be HOST_RAM/HOST_CPU, optionally followed"
+                + " by [-|*]<float> (eg. memory=HOST_RAM*.1,HOST_RAM*.2,HOST_RAM*.3,HOST_RAM*.4)."
+                + " Multiple usages are accumulated and the last value of each resource is used.")
```

Comment:
As I understand the logic as it is, this is the same behavior as all other flags where the last instance of the flag specifying a given resource is the effective value: e.g. `--test_resources=memory=1,2,3,4 --test_resources=memory=5` is effectively `--test_resources=memory=5,5,5,5`.

Slight nit here: I wouldn't use the term "accumulated" since that could imply the first flag potentially affecting the second flag. Perhaps remove the first half of the sentence or the whole sentence entirely (since that is typical flag behavior).